internal/quickstart/create_flag.go

```diff
@@ -68,8 +69,7 @@ func (m createFlagModel) View() string {
 		MarginLeft(2)
 
 	return fmt.Sprintf(
-		"Name your first feature flag (enter for default value %q):\n\n%s",
-		defaultFlagName,
```

Comment:
i removed this because the prompt felt a little long to have on one line with the input. plus with the new confirmation step they'll see the value that was used.